mysite/polls/urls.py

```diff
@@ -1,16 +1,16 @@
-from django.conf.urls import url
+from django.urls import re_path
```

Comment:
Could we test that the imports works with the previous version of Django as well? Like 2.x, and 1.9?

Reply:
Thanks you! I took your feedback and added support for older versions as well. Now it works for Django 1.11 LTS, 2.2 LTS, 3.2 TLS, and 4.0.5